sdk/communication/azure-communication-identity/samples/identity_samples.py

```diff
@@ -39,7 +38,8 @@ def get_token(self):
 
         if self.client_id is not None and self.client_secret is not None and self.tenant_id is not None:
             from azure.identity import DefaultAzureCredential
-            identity_client = CommunicationIdentityClient(self.endpoint, DefaultAzureCredential())
+            endpoint, _ = parse_connection_str(self.connection_string)
+            identity_client = CommunicationIdentityClient(endpoint, DefaultAzureCredential())
```

Comment:
Why the refactor here to parse the endpoint from the connection string?

Reply:
We didn't have an env variable in the pipeline to reference the endpoint only. One workaround would be to add the static resource endpoint as a key vault variable. Should we do this or just stick with the variable we currently have?

Reply:
I think it makes sense to do this and not add an additional environment variable to key vault.